jina/orchestrate/deployments/__init__.py

```diff
@@ -436,10 +446,12 @@ def external(self) -> bool:
     @property
     def protocol(self):
         """
-        :return: the protocol of this deployment, https or http
+        :return: the protocol of this deployment
         """
-        protocol = getattr(self.args, 'protocol', 'http')
-        return 'https' if self.is_sandbox else protocol
+        if self.is_sandbox:
+            return 'https'
```

Comment:
I couldn't find a way around this @jacobowitz

Reply:
how does the log look like for a sandbox executor then? I guess it is still printing `https://<sandboxaddress>` then?